src/tseda/vpages/map.py

```diff
@@ -61,22 +61,19 @@ def __panel__(self):
             geo=True,
             tiles=self.tiles,
             tiles_opts={"alpha": 0.5},
-            width=self.width,
-            height=self.height,
+            # height=self.height,
             size=100,
             color=color,
             tools=["wheel_zoom", "box_select", "tap", "pan", "reset"],
             fill_alpha=0.5,
             line_color="black",
-            # responsive=True,
+            responsive=True,
         )
 
     def sidebar(self):
         return pn.Card(
             self.param.tiles_selector,
-            self.param.height,
-            self.param.width,
-            collapsed=True,
+            collapsed=False,
```

Comment:
does map options need to be open or can it have default closed? maybe the user is not as likely to change this as the other options?